services/frontend-v3/src/i18n/fr_CA.json

```
@@ -291,7 +298,7 @@
   "profile.not.ex.feeder": "Je ne suis pas nominé dans le programme EX-feeder",
   "profile.private.ex.feeder": "Le résultat d'EX-feeder est privé",
   "profile.ex.feeder.title": "EX-feeder",
-  "profile.first.language": "Première langue",
+  "profile.first.language": "Langue préférée",
```

Sounds just a bit weird
```suggestion
  "profile.first.language": "Langue de préférence",
```

---

updated first language wording